examples/browser-cdn/app.js

```diff
@@ -23,8 +23,8 @@ var bugsnagClient = bugsnag({
   appVersion: '1.2.3',
 
   // Bugsnag can track the number of “sessions” that happen in your application,
-  // and calculate a crash rate for each release. This defaults to false.
-  autoCaptureSessions: true,
+  // and calculate a crash rate for each release. This defaults to true.
```

Comment:
the default value has not been updated in this PR, this comment was out of date.